.travis.yml

```diff
@@ -60,16 +66,27 @@ jobs:
       if: type != pull_request
       os: osx
 
+    - env:
+      - TEST_HDFS='true'
+      if: (type != pull_request) OR (branch =~ __TEST_HDFS__)  # Skip on PRS unless the branch contains __TEST_HDFS__
```

Comment:
Hmmm, after merging looks like this conditional isn't quite working as intended. These tests are still getting run on every PR. Looking now.